File: C++/1121.cpp
#include <iostream>
using namespace std;


int main()
{
    int x1, y1, x2, y2, dx, dy;

    cin >> x1 >> y1 >> x2 >> y2;

    dx = x1 - x2;
    if (dx < 0)
        dx = -dx;
    dy = y1 - y2;
    if (dy < 0)
        dy = -dy;

    if (x1 == x2 || y1 == y2 || dx == dy)
        cout << "YES" << endl;
    else
        cout << "NO" << endl;

    return 0;
}

File: Python/1121.py
x1, y1 = map(int, input().split())
x2, y2 = map(int, input().split())

dx = abs(x1 - x2)
dy = abs(y1 - y2)

if x1 == x2 or y1 == y2 or dx == dy:
    print('YES')
else:
    print('NO')
